### ISSUE TITLE
Missing DATASOURCE

### ISSUE THREAD

**Issue by chatiti** [lf-edge/ekuiper#2535]
I found that when using commands to create a stream, such as `bin/kuiper create stream demo (temperature string) WITH (FORM='JSON', DATASOURCE='demo')`, the manager interface (ekuiper-manager:1.8) seems unable to retrieve and update the content of DATASOURCE, but it can be preserved when created in the manager interface.

Although both methods have no impact on normal use.

**Environment**:centos

- eKuiper version (e.g. `1.3.0`): docker tag ekuiper:1.8-slim ekuiper-manager:1.8

**What happened and what you expected to happen**:
The manager interface seems to be unable to retrieve and update the content of the DATASOURCE for the stream created by the command line

**How to reproduce it (as minimally and precisely as possible)**:
Trying to create the same stream using the interface and command line will reveal

**Anything else we need to know?**:

![cmd](https://github.com/lf-edge/ekuiper/assets/79210532/84d97be4-b331-4582-a7e7-034273ae2559)
![ui](https://github.com/lf-edge/ekuiper/assets/79210532/fbf8f8f5-6f61-44b2-92aa-5b2f58b32b01)

**Comment by ngjaying** [comment]
Could you please check the return value of REST API GET streams/youStreamName?

**Comment by chatiti** [comment]
I found that streams created using two different methods both have the 'datasource' field in the return value of the `REST API GET `method. Therefore, it doesn't seem to be a backend code issue. I tried to look for frontend code but couldn't find it; perhaps it's in another project?
Thank you for your response. 😊
![Snipaste_2024-01-08_10-19-23](https://github.com/lf-edge/ekuiper/assets/79210532/f92b4407-c55e-420f-b460-2886308cf8c1)
